src/client/components/ReviewForm.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import ReviewCSS from "./UI/form.module.css";
+
+const ReviewForm = () => {
+  const [idOfMeal, setIdOfMeal] = useState("");
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [stars, setStars] = useState("");
+  const [showReviews, setShowReviews] = useState(false);
+
+  const { mealId } = useParams();
+
+  const submitReview = (e) => {
+    e.preventDefault();
+
+    const data = {
+      meal_id: idOfMeal,
+      title,
+      description,
+      stars,
+      created_date: new Date().toJSON().slice(0, 10),
+    };
+
+    fetch("api/reviews", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(data),
+    })
+      .then(() => {
+        console.log(data);
+        setTitle("");
+        setDescription("");
+        setStars("");
+      })
+      .catch((error) => {
+        alert(error);
+      })
+      .finally(alert(`You review on meal № ${mealId} was sent`));
+  };
+
+  useEffect(() => {
+    (async () => {
+      await fetch(`api/meals/${mealId}`).then((response) => response.json());
```

Comment:
Looking at it and thinking why is this needed 🤔 
You have the meal id, so you can just use it.

Reply:
I have decided to use a fetch here so that when writing a review, the user doesn’t have to write the ID of the meal by themselves, it is already included in the form. Also the alert shows the message with the correct meal ID.